widget/group.go

```diff
@@ -114,7 +114,8 @@ func (g *groupRenderer) Refresh() {
 	g.label.SetText(g.group.Text)
 	g.Layout(g.group.Size())
 
-	canvas.Refresh(g.group)
+	g.line.Refresh()
```

Comment:
If we had a `SetFillColor()` with implicit `Refresh()`, the explicit `Refresh()` could be removed here and the setter could be a no-op if the color is not being changed.

Reply:
True, but so far the canvas primitives have no setters. We could add them later if we decide that's a good idea. I can't think of a reason not to, except to avoid encouraging calling lots of functions in a row that all trigger refreshes.